src/main/java/com/tratif/pdfgen/document/DocumentBuilder.java

```diff
@@ -1,13 +1,17 @@
 package com.tratif.pdfgen.document;
 
 import com.tratif.pdfgen.helpers.CommandLineExecutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.File;
+import java.io.IOException;
 import java.nio.file.Files;
 
 public class DocumentBuilder {
 
-    private final String FILE_PREFIX = "pdfgen";
+    private final static Logger LOG = LoggerFactory.getLogger(DocumentBuilder.class);
```

Comment:
I know it's agains java naming convention, but for logger let's make an exception and name it in lowercase ("log")